packages/catalog-readmodel-writer/src/index.ts

```diff
@@ -12,6 +13,7 @@ import { handleMessageV1 } from "./consumerServiceV1.js";
 import { handleMessageV2 } from "./consumerServiceV2.js";
 
 const config = readModelWriterConfig();
+const { catalogTopic } = catalogTopicConfig();
```

Comment:
not a huge problem, but we're reading two times the env variable in these two calls. The first time to retrieve all the configurations related to the consumer and the second one to get the topic name